Wrappers/Python/ccpi/web_viewer/camera_data.py

```diff
@@ -0,0 +1,37 @@
+#
+#   Copyright 2022 STFC, United Kingdom Research and Innovation
+#
+#   Author 2022 Samuel Jones
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+from dataclasses import dataclass
+from vtkmodules.vtkRenderingCore import vtkCamera
+
+
+@dataclass(init=False)
+class CameraData:
+    position: list
+    focalPoint: list
+    viewUp: list
```

Comment:
Python type hints have been introduced in Python 3.5, I think. We should be fine with this, but is there nowhere a requirement on Python version?